src/pages/admin/company-profile/components/company-profile-social-media-tab/CompanyProfileSocialMediaTab.module.scss

```
@@ -0,0 +1,89 @@
+}
+
+.social-media-tab-header {
+    display: flex;
+    align-items: center;
+    justify-content: flex-start;
+    gap: 12px;
+}
+
+.social-media-tab-title {
+    @include type.h4-semibold;
+    color: c.$blue-900;
```

Stylelint (17.4.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)